platform/ios/src/MGLMapView.mm

```diff
@@ -1973,13 +1976,28 @@ - (void)trackGestureEvent:(NSString *)gestureID forRecognizer:(UIGestureRecogniz
 
 #pragma mark - Attribution -
 
-- (void)showAttribution:(__unused id)sender
+- (void)showAttribution:(id)sender
 {
+    BOOL shouldShowVersion = [sender isKindOfClass:[UILongPressGestureRecognizer class]];
```

Comment:
For what it’s worth, it’s pretty unlikely that a developer implementing a custom ℹ️ button will hook up a long press gesture recognizer to this method.

Reply:
This is here because I reused the existing `showAttribution:` method for the long press gesture recognizer target, so this is where we determine what the user did to trigger the attribution sheet.

Reply:
Oh, hrm — is your point that we may lose this functionality when developers roll their own? That is true, which means that maybe we should also implement one of your suggestions in https://github.com/mapbox/mapbox-gl-native/issues/6551#issuecomment-340628484.

Reply:
→ b51250464a475fb2294d8e1d8b9171e66aace40a